src/AJT/Toggl/TogglClient.php

```diff
@@ -55,12 +55,12 @@ public static function factory($config = array())
             "Content-type" => "application/json",
         ));
 
-        if(!empty($config->get('api_key'))) {
+        if($config->get('api_key')) {
             $config->set('username', $config->get('api_key'));
             $config->set('password', 'api_token');
         }
 
-        if(empty($config->get('password'))) {
+        if($config->get('password')) {
```

Comment:
This does not the same, as we go from empty to defined. 

the one above at line 58 goes from !empty to defined so has the same behaviour. 

Are you sure this is correct?